docs/gradle-plugin/index.md

```diff
@@ -91,7 +91,7 @@ Using JVM projects and Android projects differs, use the recommendations below a
 ### Kotlin JVM project
 These are projects that use Kotlin Gradle plugin.
 
-For such projects, only the [Kover default tasks](#kover-default-tasks) for generating reports are used: `koverHtmlReport`, `koverXmlReport`, `koverVerify`.
+For such projects, only the [Kover default tasks](#kover-default-tasks) for generating reports are used: `koverHtmlReport`, `koverXmlReport`, `koverIcReport`, `koverVerify`.
```

Comment:
I suggest to use `koverBinaryReport` name for such tasks, as it is at least more descriptive. Probably configuration block should be changed to `binary` as well.

Reply:
I think `koverBinaryReport` is OK